js/scripts.js

```diff
@@ -1 +1,10 @@
-﻿
\ No newline at end of file
+﻿var text = 'Velociraptor is a genus of herbivorous ceratopsid dinosaur that first appeared during the late Maastrichtian stage of the late Cretaceous period.';
+var dinosaur = 'Triceratops'.toUpperCase();
+console.log(dinosaur);
+
+
+console.log(text.replace('Velociraptor', dinosaur));
+
+var newText = text.replace('Velociraptor', dinosaur)
+
+console.log(newText.substr(0, text.length / 2));
```

Comment:
Odpowiadając na pytanie z PRa to wyglądałoby to tak:

`console.log(text.replace('Velociraptor', dinosaur).substr(0, text.length / 2));`